cli/main.go

```diff
@@ -4,12 +4,19 @@ import (
 	"context"
 	"os"
 	"os/signal"
+	"time"
 
 	"github.com/rilldata/rill/cli/cmd"
 )
 
+// These are set using -Idflags
+var Version string
+var DocsUrl = "https://docs.rilldata.com"
```

Comment:
As mentioned elsewhere, just make this a constant in the file (no need for external overrides)